185002075 Added courses page

MaxYang24 opened the pull request:
Added the courses page to show course information. It is a static page. Needs to be updates mannually.

Commit:
Footer additions

Commit:
Footer and rebase

gekerd committed:
Test yml

gekerd committed:
Delete man.yml

gekerd committed:
Update README.md

gekerd committed:
Update README.md

gekerd committed:
Update info.yml

gekerd committed:
Merge pull request #1 from cs169/gekerd-patch-1

Gekerd patch 1

gekerd committed:
Update info.yml

EthanqX committed:
finished dry run

gekerd committed:
Update README.md

gekerd committed:
Update Bluejay URL

EthanqX committed:
Merge branch 'main' of github.com:cs169/berkeley-reentry-student-program

MaxYang24 committed:
added scholarship button

EthanqX committed:
added scholarships link

EthanqX committed:
has_key? to key?

MaxYang24 committed:
Merge pull request #2 from cs169/184589769-ScholarshipLink

Scholarships link on home page

MaxYang24 committed:
merge backup

MaxYang24 committed:
merge stuff

Commit:
Rebase

MaxYang24 committed:
tried somed rspec for scholarship

Commit:
Footer additions

MaxYang24 committed:
incomplete rspec for scholarship

MaxYang24 committed:
incomplete rspec for scholar

MaxYang24 committed:
incomplete rspec for scholar

MaxYang24 committed:
finished rspec

EthanqX committed:
Merge pull request #3 from cs169/add_rspec_scholarship

Add rspec scholarship

Commit:
Footer added to home

MaxYang24 committed:
solve merge conflict

cycomachead committed:
Create pull_request_template.md

Adds a file to scaffold creating new PRs

cycomachead committed:
Clarify branch name details

cycomachead committed:
Merge pull request #5 from cs169/cycomachead-patch-1

Create pull_request_template.md

EthanqX committed:
changed layout of scholarships page

EthanqX committed:
completed reformatting

MaxYang24 committed:
Merge pull request #6 from cs169/184994821-reformat_scholarships_page

184994821 reformat scholarships page

MaxYang24 committed:
Merge branch 'main' of https://github.com/cs169/berkeley-reentry-student-program

EthanqX committed:
added nav bar

MaxYang24 committed:
Merge pull request #4 from cs169/184589774-add-footer

Footer

MaxYang24 committed:
Merge branch 'main' of https://github.com/cs169/berkeley-reentry-student-program

MaxYang24 committed:
first commit

EthanqX committed:
always expanded nav bar

MaxYang24 committed:
Merge pull request #7 from cs169/185001925-navbar_scholarships_page

185001925 navbar scholarships page

MaxYang24 committed:
added courses page

MaxYang24 committed:
added rspec

MaxYang24 committed:
added cucumber test

MaxYang24 committed:
cl fix try_1

MaxYang24 committed:
try1 rspec for courses_controller

MaxYang24 committed:
Merge branch 'main' of https://github.com/cs169/berkeley-reentry-student-program

EthanqX committed:
Merge pull request #10 from cs169/add_courses

Added courses; changed logo and main color theme

MaxYang24 committed:
added nav bars

MaxYang24 committed:
Merge branch 'main' of https://github.com/cs169/berkeley-reentry-student-program

MaxYang24 committed:
changed colors of navbars and text

MaxYang24 committed:
Merge branch 'main' into add_navbars

EthanqX committed:
Merge pull request #11 from cs169/add_navbars

Add navbars

MaxYang24 committed:
Merge branch 'main' of https://github.com/cs169/berkeley-reentry-student-program

Commit:
Resolved conflict with DS_Store

Commit:
Podcast page added

simonjov commented:
> _ No description provided. _

Please add a description for this PR for future developers to have a better idea of what specifically the high-level purpose and methods of this PR were.

simonjov reviewed (changes requested):
Again, just remove the .DS_Store files and add a description and this PR should be ready to merge!

MaxYang24 committed:
try_fix

MaxYang24 committed:
added rspec

Commit:
Test for podcast page

smanchkanti committed:
Merge pull request #13 from cs169/fix_appointment_bug

Fix appointment bug and added rspec for codecov

MaxYang24 committed:
ignore .DS_Store

MaxYang24 commented:
I've put DS.store in my gitignore file, but it still shows up. How do I delete them?

Commit:
Added rspec file for podcast

Commit:
Fixed rspec issue

gekerd committed:
Merge pull request #15 from cs169/185015924_add_podcast

add podcast page

MaxYang24 committed:
changed README.md

EthanqX committed:
Merge pull request #17 from cs169/change_readme

changed README.md

gekerd committed:
Update pages view

gekerd committed:
Merge pull request #18 from cs169/margins_update

Update pages view

cycomachead committed:
Allow Manually triggering CI run

cycomachead committed:
Merge pull request #19 from cs169/cycomachead-patch-1

Allow Manually triggering CI run

mandywong0 committed:
Ready for Heroku deployment

mandywong0 committed:
modified gemfile to specify sqlite3 version

mandywong0 committed:
removed sqlite3 and update Gemfile

mandywong0 committed:
added sqlite3 back

mandywong0 committed:
added concurrent ruby 1.3.4 to gemfile because rake db wasnt working

WinbrosXP committed:
Button shoud work

mandywong0 committed:
Re-generate Gemfile.lock to reflect production settings

mandywong0 committed:
Trigger Heroku deployment

mandywong0 committed:
just want to deploy again

mandywong0 committed:
Add x86_64-linux platform to Gemfile.lock

mandywong0 committed:
Fix dependency issues

sherher21 committed:
Merge pull request #25 from cs169/fix-appt-bttn

Button should work

sherher21 committed:
Updated readme to include new info and fixed formatting

sherher21 committed:
Added info about postgres db on heroku and notes about environmental variables

WinbrosXP committed:
Update .gitignore

WinbrosXP committed:
params[:page] was being treated as int

sherher21 committed:
Added instructions for Linux users

mandywong0 committed:
changed up wording on home page and check-in page

mandywong0 committed:
Merge pull request #35 from cs169/31-check-in-records-error

Check-in records error fixed

WinbrosXP committed:
Merge pull request #33 from cs169/mandy

Fixed Up Phrasing Inconsistencies on Home and Check-in Pages

LGTM

mandywong0 committed:
changed checkin page wording to RSP

WinbrosXP committed:
Merge pull request #36 from cs169/mandy

changed checkin page wording to RSP

LGTM

sherher21 committed:
Added codeblock around Linux instructions

KefengDuan1 committed:
Updated the restriction of SID in login.

sherher21 committed:
Merge pull request #37 from cs169/kefengduan

Updated the restriction of SID in login.

KefengDuan1 committed:
Merge pull request #26 from cs169/20-update-readme

Updated readme to include new info and fixed formatting

henwanfan committed:
Added autofill functionality for check-in reasons

- Added feature to autofill first reason ('Peer Support') for new users
- Returning users see their last used check-in reason
- Added RSpec and Cucumber tests for autofill behavior
- Updated step definitions to handle session management

henwanfan committed:
Merge branch 'main' into autofill_checkin

sherher21 committed:
Merge pull request #38 from cs169/autofill_checkin

Added autofill functionality for check-in reasons

WinbrosXP committed:
Appt page now includes cris and julio

WinbrosXP committed:
Window adjustment

sherher21 committed:
Merge pull request #39 from cs169/update-counselors

Update counselors

KefengDuan1 committed:
add rubocop workflow.

KefengDuan1 committed:
add rubocop to ReadMe.md

KefengDuan1 committed:
use rubocop to make safe edit.

KefengDuan1 committed:
unsafe edit.

KefengDuan1 committed:
correct the wrong fix due to RuboCop unsafe edit.

KefengDuan1 committed:
Fix BOM problem caused by Rubo unsafe edit.

sherher21 committed:
Added new relic gem and yml file

sherher21 committed:
Removed info.yml

sherher21 committed:
Merge pull request #43 from cs169/kefengduan

Add RuboCop Workflow.

sherher21 committed:
Merge pull request #45 from cs169/44-setup-new-relic-apm-on-heroku

Added new relic gem and yml file

sherher21 committed:
Updated badges on readme and linted code

KefengDuan1 committed:
Merge pull request #46 from cs169/readme-cleanup

Updated badges on readme and linted code

WinbrosXP committed:
Upgraded to ruby 3.0.7 (last of 3.0.*)

sherher21 committed:
Updated newrelic.yml

sherher21 committed:
Merge pull request #48 from cs169/47-remove-secret-key

Updated newrelic.yml

KefengDuan1 committed:
Update README.md

KefengDuan1 committed:
Update README.md

KefengDuan1 committed:
Add RuboCop Description in README.md

Add RuboCop Description in README.md

WinbrosXP committed:
Ruby updated to 3.1.0(First 3.1.*)

WinbrosXP committed:
Upgrade ruby to 3.1.6(last 3.1.*)

WinbrosXP committed:
Merge branch 'main' of github.com:cs169/berkeley-reentry-student-program into 40-update-ruby-rails-heroku

WinbrosXP committed:
Update ruby version file

WinbrosXP committed:
Updated ruby ver in workflow

WinbrosXP committed:
Update ruby in rubocop workflow

WinbrosXP committed:
Upgrade to ruby 3.2.0 (first 3.2.*)

WinbrosXP committed:
Upgrade to ruby 3.2.0 (first 3.2.*)

WinbrosXP committed:
Updated rubocop

WinbrosXP committed:
Upgrade to ruby 3.2.7 (last 3.2.*, last upgrade)

WinbrosXP committed:
Upgrade to rails 6.1.7.10 (last 6.*)

WinbrosXP committed:
Upgrade to rails 7.0.0 (first 7.*), cucumber 3.2, cucumber rails 2.6.1

WinbrosXP committed:
Upgrade to rails 7.0.0 (first 7.*), cucumber 3.2, cucumber rails 2.6.1

WinbrosXP committed:
Upgrade to rails 7.0.0 (first 7.*), cucumber 3.2, cucumber rails 2.6.1

WinbrosXP committed:
Upgraded rails to 7.0.8, other updates in gemfile for rails compatibility

WinbrosXP committed:
db update

WinbrosXP committed:
Trying to diagnose Logger error

WinbrosXP committed:
Trying to diagnose Logger error

WinbrosXP committed:
Trying to diagnose Logger error

mandywong0 committed:
updated content on scholarship and courses pages. updated UI across whole website. increased contrast in colors.

mandywong0 committed:
reversed gitignore changes

sherher21 committed:
Merge pull request #60 from cs169/mandy

Updated Scholarship and Courses content & Made UI changes for Accessibility

henwanfan committed:
Replace Counselor Appointment with Drop-in Advising in Checkin dropdown

WinbrosXP committed:
Merge pull request #61 from cs169/update-checkin-dropdown-57

Replace Counselor Appointment with Drop-in Advising in Checkin dropdown

WinbrosXP committed:
New spec/requests folder for converting controller to request specs

WinbrosXP committed:
Request test for admin functions

sherher21 committed:
Added working canvas authentication

sherher21 committed:
Removed storing token in database and updated root page

sherher21 committed:
Restored google login

sherher21 committed:
Added rspec tests for bcourse authentication

sherher21 committed:
Added failing cucumber tests

WinbrosXP committed:
Conversions to request tests for admins and checkin

sherher21 committed:
Linter

sherher21 committed:
Merge pull request #62 from cs169/bcourse-authentication

Bcourse authentication

WinbrosXP committed:
Removed checkin_controller_spec and converted to request test

WinbrosXP committed:
Fixed issue with cucumber tests

WinbrosXP committed:
Merge branch 'main' into ruby-updates

WinbrosXP committed:
Rubocop deprecation fix

WinbrosXP committed:
Merge branch 'ruby-updates' of github.com:cs169/berkeley-reentry-student-program into ruby-updates
rubocop.yml update

henwanfan committed:
Merge pull request #69 from cs169/ruby-updates

Ruby updates

henwanfan committed:
Implement Event model and basic controller functionality (#63)

mandywong0 committed:
updated ruby version after mergint from main

mandywong0 committed:
fixed merge conflicts?

mandywong0 committed:
fixed merge conflicts?

mandywong0 committed:
removed trailing period on accessibility.feature

KefengDuan1 committed:
fix the failure of cucumber test.

henwanfan committed:
Fix RuboCop issues in Event model

mandywong0 committed:
Recreate Gemfile.lock

mandywong0 committed:
updated gemfile.lock

mandywong0 committed:
disabled webmock if selenium chrome in capybara.rb

KefengDuan1 committed:
revert wrong fix.

WinbrosXP committed:
Added partial for navbar, still testing

WinbrosXP committed:
Removed duplicate navbars from views (hopefully all)

henwanfan committed:
Merge pull request #73 from cs169/kefengduan

revert wrong fix.

henwanfan committed:
Merge pull request #71 from cs169/feature/63-event-model-controller

Implement Event model and basic controller functionality (#63)

mandywong0 committed:
fixed webmock and capybara conflicts

KefengDuan1 committed:
support loading scholarships from database.

KefengDuan1 committed:
update README.md for database.

KefengDuan1 committed:
fixes made by RuboCop

henwanfan committed:
Merge pull request #72 from cs169/mandy

Added Accessibility Tests and Fixed UI Accordingly

WinbrosXP committed:
Merge with main

WinbrosXP committed:
Added events to navbar

WinbrosXP committed:
Fixing events link

WinbrosXP committed:
Fixing events link

WinbrosXP committed:
Fixing events link

WinbrosXP committed:
Fixing accessibility

WinbrosXP committed:
Started cucumber scenario for events

WinbrosXP committed:
Started cucumber scenario for events

WinbrosXP committed:
Fix rubocop issue

WinbrosXP committed:
Merge pull request #74 from cs169/kefengduan

Feature: Load scholarships from databases.

mandywong0 committed:
Merge branch 'main' into 52-navbar

mandywong0 committed:
Merge pull request #75 from cs169/52-navbar

Navbar

henwanfan committed:
Implement Events page with upcoming and past events sections

henwanfan committed:
removed events from database as it was not working, using rails console to test instead

henwanfan committed:
Add events page with upcoming and past events display

henwanfan committed:
Merge pull request #76 from cs169/feature/64-events-page-ui

Implemented Events page UI

WinbrosXP committed:
Merge changes from main

WinbrosXP committed:
Merged changes from main

WinbrosXP committed:
Fix rubocop offenses

WinbrosXP committed:
Removed controller tests that have been converted

WinbrosXP committed:
Trigger deploy on new stack

sherher21 committed:
Added model Advisor

KefengDuan1 committed:
support edit scholarships.

KefengDuan1 committed:
rubocop fix.

KefengDuan1 committed:
feature:remove scholarships.

KefengDuan1 committed:
feature: add_scholarships.

KefengDuan1 committed:
fix bugs in deleting scholarships

KefengDuan1 committed:
RuboCop fixes.

sherher21 committed:
Added advisor admin feature

mandywong0 committed:
added admin views for event index and event create. implemented create event functionality

mandywong0 committed:
fixed rubocup failures

henwanfan committed:
Merge pull request #80 from cs169/kefengduan

support edit scholarships.

henwanfan committed:
Merge pull request #81 from cs169/delete_scholarships

feature: delete scholarships

henwanfan committed:
Merge pull request #82 from cs169/add_scholarships

feature: Add scholarships

henwanfan committed:
Merge branch 'main' into 30-add-function-to-addremove-counselors-and-their-calendar-on-appointment-page

henwanfan committed:
Merge pull request #83 from cs169/30-add-function-to-addremove-counselors-and-their-calendar-on-appointment-page

Add admin feature to manage advisors

sherher21 committed:
Merge branch 'main' into event_admin

sherher21 committed:
Merge pull request #84 from cs169/event_admin

Added admin view for event management & implemented basic functionality for creating new events

WinbrosXP committed:
Merge branch 'main' into 40-update-ruby-rails-heroku

WinbrosXP committed:
Merge branch 'main' into 40-update-ruby-rails-heroku

WinbrosXP committed:
Merge branch '40-update-ruby-rails-heroku' of github.com:cs169/berkeley-reentry-student-program into 40-update-ruby-rails-heroku

WinbrosXP committed:
Rubocop fix

WinbrosXP committed:
Removes remaining controller tests, removes failing cucumber steps

WinbrosXP committed:
Fixing webmock issue

WinbrosXP committed:
Rubocop fixes

sherher21 committed:
Merge pull request #79 from cs169/40-update-ruby-rails-heroku

40 update ruby rails heroku

sherher21 committed:
Merge branch 'main' into interim-branch-update
